api/server.py

```diff
@@ -372,6 +374,49 @@ def delete_all_projects():
     # Number of prizes they can choose per challenge
     # ProjectID that won the challenge
 
+@app.route('/api/import-challenges')
+def import_challenges():
+    devpost_url = "https://bitcamp2018.devpost.com/submissions"
+    companies = mongo.db.companies
+
+    prize_list = get_challenges(devpost_url)
+    company_list = []
+    company_names = list(set([prize[1] for prize in prize_list]))
+    
+    for company_name in company_names:
+        challenge_info = [[prize[0],prize[2]] for prize in prize_list
+                         if prize[1] == company_name]
+
+
+        #Autogenerate access_code
```

Comment:
Awesome. Great job finding this piece and adding it in - loving your thoroughness!